_IBC/nolus-neutron.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
```

Comment:
not following alphabetical order. Should be Neutron<>Nolus

Reply:
Not only the file names, but also, chain_1 and chain_2 must follow alphabetical order, and the corresponding channels must also be reversed to reflect the order change.

Reply:
Oh, right. Just pushed the changes, should be good now.